scripts/tests/localsetup_tests.py

```diff
@@ -8,14 +8,14 @@
 
 from .. import compose
 
-from ..compose import (OpbeansPython, OpbeansRum, OpbeansGo, OpbeansJava,
-                       OpbeansNode, OpbeansRuby, OpbeansLoadGenerator, OpbeansDotnet)
+from scripts.elastic_stack import ApmServer, Elasticsearch, Kibana
 
-from ..compose import (ApmServer, Kibana, Elasticsearch)
+from scripts.aux_services import Postgres, Redis
 
-from ..compose import (Postgres, Redis)
-
-from ..compose import LocalSetup, discover_services, parse_version, OpbeansService
+from ..compose import LocalSetup
+from scripts.opbeans import OpbeansService, OpbeansDotnet, OpbeansGo, OpbeansJava, OpbeansNode, OpbeansPython, \
```

Comment:
typo?

```suggestion
from scripts.opbeans import OpbeansService, OpbeansDotnet, OpbeansGo, OpbeansJava, OpbeansNode, OpbeansPython, 
```

Reply:
@v1v the `\` is needed because the next line is a continuation of the import statement. I myself prefer to use parentheses:

    from scripts.opbeans import (
        X, Y, Z,
        A, B, C,
    )

Reply:
got it